package.json

```diff
@@ -7,16 +7,22 @@
     "@emotion/styled": "^11.10.4",
     "@mui/icons-material": "^5.10.9",
     "@mui/material": "^5.10.10",
+    "@reduxjs/toolkit": "^1.8.6",
```

Comment:
On your readme:

- [ ] Your readme does not correspond to the Microverse Readme Template (your backend does!).  Please, go [here](https://github.com/microverseinc/readme-template/blob/master/README.md) and use this template to craft your professional readme.  Please, don't forget the screenshot!  Your design is beautiful let's show it off 💪